src/main/java/org/jenkinsci/maven/plugins/hpi/DependencyFilterProvider.java

```diff
@@ -0,0 +1,23 @@
+package org.jenkinsci.maven.plugins.hpi;
+
+import java.util.Map;
+
+import org.apache.maven.artifact.Artifact;
+import org.eclipse.aether.graph.DependencyFilter;
+
+public class DependencyFilterProvider {
+    Map<String, DependencyFilterFactory> FACTORIES = 
```

Comment:
Please remove the trailing white space on this line.  It may distract future reviewers.
```suggestion
    Map<String, DependencyFilterFactory> FACTORIES =
```